package-lock.json

```diff
@@ -13316,7 +13316,15 @@
       "integrity": "sha512-VMtK//85QJomhk3cXOCksNwOYaw1KWnYTS37GYGgyf7A3ajdBoPGhaJuJWAH2S2kq8GZeXkdKn+3Mfmgy11cVw==",
       "dev": true,
       "requires": {
-        "icu4c-data": "0.64.2"
+        "icu4c-data": "0.65.2"
```

Comment:
Can we remove these changes? They're not required for this PR.

Reply:
Fixed it. Hope this fixes everything finally.  But I was thinking that changes in `package-lock.json` are bound to happen. Isn't there some way to prevent it. Every time running `npm install` will result in change in `package-lock.json`

Reply:
They will, we just don't commit them with every PR. Thanks for reverting :)